plugins/importexport/native/filter/NativeExportFilter.inc.php

```diff
@@ -24,6 +29,46 @@ function __construct($filterGroup) {
 		parent::__construct($filterGroup);
 	}
 
+	/**
+	 * Set no validation option
+	 * @param $noValidation boolean
+	 */
+	function setNoValidation($noValidation) {
+		$this->_noValidation = $noValidation;
+	}
+
+	/**
+	 * Get no validation option
+	 * @return boolean true|null
+	 */
+	function getNoValidation() {
+		return $this->_noValidation;
+	}
+
+	//
+	// Public methods
+	//
+	/**
+	 * @copydoc Filter::supports()
+	 */
+	function supports(&$input, &$output) {
```

Comment:
Yes, I think so -- it sets the validation strategy to none, if the noValidation option is used...